### ISSUE TITLE
Revisit Algolia indexing and search configuration

### ISSUE THREAD

**Issue by cwjohnston** [sensu/sensu-docs#2018]
## This is an issue with:

- [ ] Bug (site functionality or styling)
- [ ] Errata (fix needed for doc content)
- [ ] New content (guide wanted)
- [x] Update (Add missing or refresh existing content)
- [ ] Enhancement (add new site functionality)

## Expected Behavior

Search will focus on the “latest” version of the docs, presenting the most up to date information and reducing duplicate results.

## Current Behavior

Search shows duplicate results for many queries.

## Possible Solution

Revise parameters passed to algolia in https://github.com/sensu/sensu-docs/blob/master/layouts/partials/footer_js.html

## Context

Once we merge #2015 we will have a much more straightforward story about how “latest” works, should enable us to improve the usefulness of search.

* Browser(s) and version(s): 
* OS:

**Comment by hillaryfraley** [comment]
Would we replace `5.16` with `latest` in this code?

```
algoliaOptions: { 'facetFilters': [[{{ if isset .Page.Params "product" }}'tags:{{ .Section }}'{{ else }}'tags:sensu-go','tags:plugins', 'tags:sensu-core', 'tags:sensu-enterprise', 'tags:sensu-enterprise-dashboard', 'tags:uchiwa'{{end}}], [{{ if isset .Page.Params "product" }}'version:{{ .Page.Params.version }}'{{ else }}'version:5.16','version:1.0','version:2.16','version:1.8','version:3.6'{{end}}]]},
```

**Comment by hillaryfraley** [comment]
I think search isn't working on 5.16. Every search produces "No results found for x"

**Comment by cwjohnston** [comment]
We tried changing the default version to 'latest' for all products in #2047 but that doesn't seem to be having the desired effect. @asachs01 will start up a conversation with Algolia team to investigate with @rnevius copied on the message.

**Comment by cwjohnston** [comment]
Seems like we are using the content of each anchor link in the version selector dropdown to generate version selectors in algolia: https://github.com/algolia/docsearch-configs/blob/a43d72b0d351f5c65324f36f24bad7d8b3185180/configs/sensu.json#L45-L56

Since we've appended ` (latest)` to the content of the version for the `latest` version, I think this means we need to either

* modify the version provided to algolia search API to include the ` (latest)` text.
Or 
* rework the algolia search configuration to ignore ` (latest)` text when building selectors as part of crawling the site

**Comment by cwjohnston** [comment]
Maybe we can access site params from config.toml to compare the value of `site.params.products.$PRODUCT.latest` with `page.params.version` and render the algolia search parameters accordingly?

**Comment by hillaryfraley** [comment]
Closed by #2049
